### DISCUSSION TITLE
How to start with nuxt-directus?

### DISCUSSION THREAD

**Discussion by xnacly** [Intevel/nuxt-directus#49] [Q&A] [answered]
I wanted to try it out using nuxt. 
Foremost, what does and doesn't your lib support, and how can it help me integrate directus into my nuxt app? 
Second, how exactly do i get started with directus in nuxt and what steps would you recommend me to take to fully grasp directus and its usefulness. 
And thirdly what are the common best practices associated with directus and nuxt. 

Loooking forward to your response 😄

**Comment by Intevel** [comment, marked as answer]
Hey @xnacly, 
With `nuxt-directus` you can easily implement the authentication and also the handling of collection items. When you start, however, you should first deal with Nuxt 3 and Directus. 

Create new projects with both and feel your way with the nuxt-directus documentation.
